bin/elasticity-flexture-L0/3_modeling_plate_bending.ipynb

```diff
@@ -0,0 +1,254 @@
+{
```

Comment:
<p>Capitalize "t" in the sentence starting with "<strong>t</strong>hese loads"</p><p> </p><p>Use "Kyr", "Ka", or "thousand of year" instead of "K" after "90"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geodynamics-hubzero/gem-notebooks/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>